src/host.rs

```diff
@@ -85,7 +85,7 @@ impl Host {
     pub(crate) fn receive_from_network(&mut self, envelope: Envelope) -> Result<(), Protocol> {
         let Envelope { src, dst, message } = envelope;
 
-        trace!(?dst, ?src, protocol = %message, "Delivered");
+        tracing::trace!(target: TRACING_TARGET, ?dst, ?src, protocol = %message, "Delivered");
```

Comment:
Isnt the target already set since the crate name is `turmoil`?

Reply:
It is the full mod path, which isn't major but it's pretty nice to just see turmoil on all the events.

Reply:
Interesting, @davidbarsky I wonder what you think

Reply:
Per #49 I assume everything that went to `trace` in this change would be a span. I don't like seeing the internal mod path in the traces, which is why I use a constant. It's not really a trace for the crate, rather it's a network trace that we happen to use tracing for. I don't feel too strongly though, just personal preference. Long term, I think we want a subscriber included that does nice things like write in different formats, and to different sinks like a file.